src/pybind/mgr/dashboard/frontend/src/app/ceph/shared/feedback/feedback.component.html

```diff
@@ -0,0 +1,120 @@
+<cd-modal [modalRef]="activeModal">
+  <ng-container class="modal-title"
```

Comment:
`<ng-container>` is meant to be used as placeholder for structural directives (*ngIf, *ngFor, ...) when you do not want to add a new element just for this, which does not seem to be the case. 
```suggestion
  <div class="modal-title"
```